trio/_core/_generated_run.py

```diff
@@ -39,7 +45,7 @@ def current_statistics():
         raise RuntimeError("must be called from async context")
 
 
-def current_time():
+def current_time() ->float:
```

Comment:
`() -> float:`

Reply:
This file is generated and not edited manually, the other annotations will appear when we annotate the classes elsewhere.